### PR TITLE
subscribe functionality & user's subscriptions on home page

### PR DESCRIPTION
**Pull request by sojeri**

### CODE REVIEW

File: app/views/welcome/_feed.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<% if current_user.avatar %>
+  <%= image_tag current_user.avatar %>
+<% end %>
+
+<h2 class="text-center"><%= current_user.name %>'s feed</h2>
+
+<% if posts.nil? || posts.empty? %>
+  <p>
+    You haven't subscribed to any feeds yet. Find some interesting feeds to
```
Comment (Brennaleker) [inline app/views/welcome/_feed.html.erb:9]: Awesome!

**Comment by Brennaleker** [comment]
Looks great to me :)  :+1:

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
 class WelcomeController < ApplicationController
-  def index;end
+  def index
+    # reset_session # TODO: remove this before final PR. keep until no more rake db:reset
+    if current_user
+      @posts = current_user.posts.chronological.only_thirty
```
Comment (vikshab) [inline app/controllers/welcome_controller.rb]: Maybe we can make it shorter, looks like a long chain of methods.

File: app/models/post.rb [hunk 3]
```diff
@@ -1,4 +1,11 @@
 class Post < ActiveRecord::Base
-  # Associations--------------------------------------------
+  # Associations ---------------------------------------------------------------
   belongs_to :feed
+
+  # Validations ----------------------------------------------------------------
+  validates_presence_of :feed_id
+
+  # Scopes ---------------------------------------------------------------------
+  scope :chronological, -> { order("date_posted DESC")}
```
Comment (vikshab) [inline app/models/post.rb]: maybe combine and make just one scope?

**Pull request merged**